CMakeLists.txt

```diff
@@ -138,6 +138,8 @@ ADD_COMPILE_DEFINITIONS("XNN_ENABLE_MEMOPT=$<BOOL:${XNNPACK_ENABLE_MEMOPT}>")
 ADD_COMPILE_DEFINITIONS("XNN_ENABLE_SPARSE=$<BOOL:${XNNPACK_ENABLE_SPARSE}>")
 ADD_COMPILE_DEFINITIONS("XNN_ENABLE_GEMM_M_SPECIALIZATION=$<BOOL:${XNNPACK_ENABLE_GEMM_M_SPECIALIZATION}>")
 ADD_COMPILE_DEFINITIONS("XNN_ENABLE_DWCONV_MULTIPASS=$<BOOL:${XNNPACK_ENABLE_DWCONV_MULTIPASS}>")
+ADD_COMPILE_DEFINITIONS("XNN_LOG_LEVEL=2")
```

Comment:
Unrelated debugging change?

Reply:
I initially added this to remove the error: 

.../XNNPACK/src/packing.c:15:
.../XNNPACK/src/xnnpack/log.h:20:4: error: "Undefined XNN_LOG_LEVEL"
  #error "Undefined XNN_LOG_LEVEL"
   ^
1 error generated.

Although it worked, I knew this may not be the best solution because I was getting 'XNN_LOG_LEVEL' was redefined in some parts of the build. Any suggestions?